### DISCUSSION TITLE
Issues after upgrade to v1.1.4

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#6718] [Help]
I am facing issues after upgrade for the last two versions. This time the REST queries fail with the following error `Invalid JSON for request body`

If there a way to make the apps that were working continue to run with the last version of Budi till upgraded? I don't see the upgrade button on the app either now.

**Comment by SantoshSrinivas79** [comment]
The request isn't even making its way to the rest endpoint. The request body after upgrade looks like this.
Is this the new way to access Binding variables?

<img width="455" alt="Screenshot 2022-07-15 at 7 30 50 AM" src="https://user-images.githubusercontent.com/1036163/179131789-804ccbe7-2830-4e5c-a821-9227fe8cd1b4.png">

Its also weird that the upgrade changes are not consistent. Here is another query where the changes dont seem to be consistent in format .... few params sometimes show prefixed with `Binding` and the others dont.

<img width="398" alt="Screenshot 2022-07-15 at 7 32 49 AM" src="https://user-images.githubusercontent.com/1036163/179132106-04c32cb0-e946-492a-a6a2-26600d1160ea.png">

**Reply by deanhannigan** [reply]
Hey @SantoshSrinivas79, thanks for getting in touch. All bindings in the query body are given contextual prefixes on load.
The intention was to make them easily identifiable to users given the many contexts from which they can be applied. We may need to review the UX for JSON body bindings to make things a bit clearer.
Either `{{tweet_id}} `or `{{Binding.tweet_id}}` can be used when constructing your query body, but it will always be saved in the DB as `{{tweet_id}}`. 

Regarding your particular issue here, the binding name `{{user}}` is currently reserved and it will not map to your current binding value. If you change the name of the binding to e.g. `{{my_user}}` it should work as expected. Your second example above should work as expected with this modification.

The recent changes relating to user bindings and the discussion around them are also covered further here.
https://github.com/Budibase/budibase/pull/6315

If you have any further queries, let us know.
